packages/design-system/src/components/web-components/__snapshots__/ds-alert.test.tsx.snap

```diff
@@ -63,9 +63,8 @@ exports[`Alert renders alert 1`] = `
     id="static-id"
   >
     <div
-      aria-labelledby="static-id__a11y-label"
+      aria-labelledby="alert--1__a11y-label"
```

Comment:
Do you know why this changed? I don't see code that actually defines the `root-id`, but I also don't see that it was removed in this branch. Like I get why the static id isn't present, but why was it ever static before?

Reply:
i noticed the static id wasn't set and figured this was a fix but no clue why it changed.